### PR TITLE
Pull Request, which will contain all changes from ODF 1.3  CSD03 to ODF 1.3 CS02

### PR DESCRIPTION
**Pull request by svanteschubert**
Started with a single issue and will add fixes incrementally. Flagging this PullRequest therefore as a draft, until it is ready!
**NOTE**: Using a label as the "draft pull request feature" exists only for GitHub Team & Enterprise ;-)

### CODE REVIEW

**Comment by svanteschubert** [comment]
Please note, I have fixed first the ODT, by using JEdit with archive PlugIn to access, edit and save the content.xml file within the ZIP.
Afterwards, I have been using an unzip batch file to unpack the ODT into the folder.
Therefore, if the change of the unzipped content.xml is correct, so is the ODT :-)

**Comment by svanteschubert** [comment]
At this stage, decided to better stick to the TC-agreed wording from the former issue: 
https://issues.oasis-open.org/browse/OFFICE-3873
Instead of following the wording recently suggested by Patrick:
https://lists.oasis-open.org/archives/office/202010/msg00016.html

**Comment by svanteschubert** [comment]
Changes made so far:

1. Updated the wording according to https://issues.oasis-open.org/browse/OFFICE-3873
2. Updated IDs of deliverables references in the ODT (exchanged 'CSD03' to 'CS02' and 'CS01' to 'CSD03') 
3. Fixing in part 3 the 'broken' link from 'http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.49.6093' to 'https://ctan.mirror.norbert-ruehl.de/biblio/bibtex/base/btxdoc.pdf'
4. Fixing missing whitespace after MathML entities in HTML
5. Generated new HTML & PDF documents

**Review by mistmist — COMMENTED** [review]
the .owl files need fixing

File: src/main/resources/odf1.3/OpenDocument-v1.3-metadata.owl [hunk 1]
```diff
@@ -1,62 +1,59 @@
...
-     Source: https://docs.oasis-open.org/office/OpenDocument/v1.3/csd03/schemas/
+     Source: https://docs.oasis-open.org/office/OpenDocument/v1.3/cs02/schemas/
      Latest stage of specification: https://docs.oasis-open.org/office/OpenDocument/v1.3/OpenDocument-v1.3-part1-introduction.html
      TC IPR Statement: https://www.oasis-open.org/committees/office/ipr.php 
 
         Open Document Format for Office Applications (OpenDocument) Version 1.3
-        OWL Schema Metadata Manifest Description
-        OpenDocument-v1.3-metadata.owl
+        OWL Package Metadata Manifest Description
+        OpenDocument-v1.3-package-metadata.owl
```
[the remark this answers is not in the document]
Comment (svanteschubert) [reply src/main/resources/odf1.3/OpenDocument-v1.3-metadata.owl]: Thank you, the complete file seemed wrong due to a copy/paste error of mine! :-)

File: docs/odf1.3/cs02/schemas/OpenDocument-v1.3-metadata.owl [hunk 2]
```diff
@@ -1,62 +1,59 @@
...
-     Source: https://docs.oasis-open.org/office/OpenDocument/v1.3/csd03/schemas/
+     Source: https://docs.oasis-open.org/office/OpenDocument/v1.3/cs02/schemas/
      Latest stage of specification: https://docs.oasis-open.org/office/OpenDocument/v1.3/OpenDocument-v1.3-part1-introduction.html
      TC IPR Statement: https://www.oasis-open.org/committees/office/ipr.php 
 
         Open Document Format for Office Applications (OpenDocument) Version 1.3
-        OWL Schema Metadata Manifest Description
-        OpenDocument-v1.3-metadata.owl
+        OWL Package Metadata Manifest Description
+        OpenDocument-v1.3-package-metadata.owl
```
[the remark this answers is not in the document]
Comment (svanteschubert) [reply docs/odf1.3/cs02/schemas/OpenDocument-v1.3-metadata.owl]: Thank you, the complete file seemed wrong due to a copy/paste error of mine! :-)

**Comment by mistmist** [comment]
dont merge yet, the files still contain "Draft 03"...

**Comment by svanteschubert** [comment]
The final summary of all (non-material) changes: 

* Updated the wording according to https://issues.oasis-open.org/browse/OFFICE-3873
* Updated IDs of deliverables references 
    * in all file names and added a schema directory for RNG, OWL files
    * in the ODT (exchanged 'Draft 03' to '02', 'CSD03' to 'CS02' and 'CS01' to 'CSD03') 
    * in the RNG/RNG-HTML 
    * in the OWL files 
* Fixing in part 3 the 'broken' link from 'http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.49.6093' to 'https://ctan.mirror.norbert-ruehl.de/biblio/bibtex/base/btxdoc.pdf'
* Fixing ODF 1.2 XML manually by removing ODF 1.3 attributes to become valid ODF 1.2
* Fixing missing whitespace after MathML entities in HTML
* Generated new HTML & PDF documents

**Pull request merged**